test/common/EnvVars.cpp

```diff
@@ -116,6 +151,13 @@ namespace RcclUnitTesting
     // Total number of reduction ops
     int numOps = ncclNumOps;
 
+    bool cpxMode = false;
```

Comment:
For consistency, please rename this bool as isCpxMode

Reply:
I have raised the pull request with the requested change: https://github.com/ROCm/rccl/pull/1327/files